arch/eventloop_posix_tcp.c

```diff
@@ -11,19 +11,21 @@
 #include "eventloop_common.h"
 
 /* Configuration parameters */
-#define TCP_PARAMETERSSIZE 5
+#define TCP_PARAMETERSSIZE 6
 #define TCP_PARAMINDEX_RECVBUF 0
 #define TCP_PARAMINDEX_ADDR 1
 #define TCP_PARAMINDEX_PORT 2
 #define TCP_PARAMINDEX_LISTEN 3
 #define TCP_PARAMINDEX_VALIDATE 4
+#define TCP_PARAMINDEX_REUSEADDR 5
 
 static UA_KeyValueRestriction TCPConfigParameters[TCP_PARAMETERSSIZE] = {
     {{0, UA_STRING_STATIC("recv-bufsize")}, &UA_TYPES[UA_TYPES_UINT32], false, true, false},
     {{0, UA_STRING_STATIC("address")}, &UA_TYPES[UA_TYPES_STRING], false, true, true},
     {{0, UA_STRING_STATIC("port")}, &UA_TYPES[UA_TYPES_UINT16], true, true, false},
     {{0, UA_STRING_STATIC("listen")}, &UA_TYPES[UA_TYPES_BOOLEAN], false, true, false},
-    {{0, UA_STRING_STATIC("validate")}, &UA_TYPES[UA_TYPES_BOOLEAN], false, true, false}
+    {{0, UA_STRING_STATIC("validate")}, &UA_TYPES[UA_TYPES_BOOLEAN], false, true, false},
+    {{0, UA_STRING_STATIC("reuseaddr")}, &UA_TYPES[UA_TYPES_BOOLEAN], false, true, false}
```

Comment:
The UDP connection manager has a pre-rexisting option called `reuse`.
Let's do it identical everywhere. --> reuse.